packages/core/protocols/src/proto/dxos/config.proto

```diff
@@ -83,7 +84,12 @@ message Runtime {
       optional string path = 4;                   // TODO(burdon): Rename data_path.
     }
 
-    optional string debug = 1 [(env) = "DEBUG"];  // TODO(burdon): Change to LOG.
+    message Log {
+      optional string filter = 1 [(env) = "LOG_FILTER"];
+      optional string prefix = 2 [(env) = "LOG_PREFIX"];
+    }
+
+    optional Log log = 1;
```

Comment:
I realize this isn't backward compatible but figured we might be able to get away with it since it's not really being used currently.